binderhub/static/index.js

```diff
@@ -225,6 +206,19 @@ $(function(){
         return false;
     });
 
+    $('#toggle-badge-snippet').on('click', function() {
+        var badgeSnippets = $('#badge-snippets');
+        if (badgeSnippets.hasClass('hidden')) {
+            badgeSnippets.removeClass('hidden');
+            snippetsVisible = true;
```

Comment:
`snippetsVisible` seem to be an unused variable. And if you remove it, you can switch to `.toggleClass('hidden')` (http://api.jquery.com/toggleclass/)

Reply:
Also just FYI most of jQuery functionality are now part of browsers (but with less magic, in browser). If we want to get rid of jQ, we could use `document.getElementById('badge-snippets').classList.toggle('hidden')` which also has the advantage of returning true/false to know wether the class was added or removed :-) though the jQuery is more redable IMHO and way more compact !

Reply:
removed that variable, I was mimicking the style of logs but I agree we don't need unnecessary variables in there. I'm gonna keep this as jq for now, perhaps when we refactor the UI we can decide if we wanna switch up the code

Reply:
> I'm gonna keep this as jq for now, perhaps when we refactor the UI we can decide if we wanna switch up the code

Sounds good.